src/apps/components/InstalledAppListRow/InstalledAppListRow.test.tsx

```diff
@@ -4,13 +4,14 @@ import { InstalledApp } from "@dashboard/apps/types";
 import { getAppsConfig } from "@dashboard/config";
 import Wrapper from "@test/wrapper";
 import { render, screen } from "@testing-library/react";
-import React from "react";
 import { MemoryRouter as Router } from "react-router-dom";
 
 import InstalledAppListRow from "./InstalledAppListRow";
 
 const Component = ({ data, context }: { data: InstalledApp; context: AppListContextValues }) => (
   <Wrapper>
+    {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+    {/* @ts-ignore */}
```

Comment:
suggestion: we should use `@ts-expect-error` instead of `@ts-ignore`, because if for some reason we don't have TypeScript error anymore it will be shown as error so that we can remove this comment. You also don't need to disable ESLint rule for that (probably)